generated/nisync/nisync.proto

```diff
@@ -0,0 +1,45 @@
+
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Proto file for the NI-SYNC Metadata
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ni.grpc.sync";
+option java_outer_classname = "NiSync";
+option csharp_namespace = "NationalInstruments.Grpc.Sync";
+
+package nisync_grpc;
+
+import "session.proto";
+
+service NiSync {
+  rpc Init(InitRequest) returns (InitResponse);
+  rpc Close(CloseRequest) returns (CloseResponse);
+}
+
+enum NiSyncAttributes {
+  NISYNC_ATTRIBUTE_UNSPECIFIED = 0;
+}
+
+message InitRequest {
+  string resource_name = 1;
+  bool _i_d_query = 2;
```

Comment:
[FYI] This is generating like `_i_d_query` because we're expecting the field name to be in camelCase in the metadata and we translate that to snake case here. If you want, you could change the field name in the metadata you're adding to be idQuery so it generates here and everywhere else as `id_query` instead.

Reply:
Also, if you do end up modifying the metadata files you're importing consider adding a CHANGES.md file to the metadata folder. We found it useful for tracking manual updates to the metadata we made and why. Here's niscope's [CHANGES.md](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/niscope/CHANGES.md) file as an example.

Reply:
Updated idQuery and documented in CHANGES.md